src/components/ToolTips/ToolTips.tsx

```diff
@@ -179,6 +181,10 @@ export function getTip(tipType: string) {
             return render(FM.TOOLTIP_ACTION_CARD_TITLE, [FM.TOOLTIP_ACTION_CARD])
         case TipType.ACTION_END_SESSION:
             return render(FM.TOOLTIP_ACTION_END_SESSION_TITLE, [FM.TOOLTIP_ACTION_END_SESSION])
+        case TipType.ACTION_SET_ENTITY:
+            return render(FM.TOOLTIP_ACTION_SET_ENTITY_TITLE, [FM.TOOLTIP_ACTION_SET_ENTITY])
+        case TipType.ACTION_ENTITY_VALUE:
+            return render(FM.TOOLTIP_ACTION_SET_ENTITY_TITLE, [FM.TOOLTIP_ACTION_SET_ENTITY])
```

Comment:
Not sure what context these are used in, but these two different tooltips seem to be showing the save values?  Is this work TBD?

Reply:
Yea, the tool tips don't really add value.  It's trying to explain what an Entity dropdown is. I would think it's obvious it is a list of entities, but I still added it since we have pattern of doing that for all our fields.

I agree it should add more detail but I did think it's probably best to keep these fields (tooltips) the same. I thought a higher level explanation of what set entity actions do such as explaining how they're used for context sensitive situations when label value alone is ambiguous.

What do you think we should put in the panel?